src/ResourceManager/KeyVault/Commands.KeyVault/help/Set-AzureKeyVaultKeyAttribute.md

```diff
@@ -1,4 +1,4 @@
-﻿---
```

Comment:
@maddieclayton same comment about this call being a [`PATCH`](https://github.com/Azure/azure-sdk-for-net/blob/psSdkJson6/src/SDKs/KeyVault/dataPlane/Microsoft.Azure.KeyVault/Generated/KeyVaultClient.cs#L963), so we should change the verb to `Update`